app/templates/editor/edit.hbs

```diff
@@ -50,9 +50,11 @@
                 setEditor=(action "setEditor")
                 menuIsOpen=(action "editorMenuIsOpen")
                 menuIsClosed=(action "editorMenuIsClosed")
+                wordCount=wordCount
```

Comment:
I think we should be sticking to DDAU rather than relying on implicitly bound immutable data. There's no need to pass a word count into the `gh-koenig` component so it should be possible to pass an action in here instead.